src/WebJobs.Script/Description/Script/ScriptInvocationContext.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Script.Description.Script
```

Comment:
This namespace doesn't look right. This existing Description\Script folder was for all the existing out of proj "script" languages.

The 2 new types you added here seem instad to belong to the Description\Worker folder, since the types are used by the WorkerLanguageInvoker?

Reply:
Also can you make these types internal? In general, check the visibility of the new types.

Reply:
Yep, should be internal, and that's a weird namespace. I'll smooth over a lot of those pieces.

Reply:
@mamaso